UnitTests/MappersTest/UserMapperTests.cs

```diff
@@ -181,5 +182,65 @@ public void LeaveReview()
             }
             Assert.True(successRiderToDriverFeedback);
         }
+
+        /**
+        * Test whether we are able to successfully get the GSD of a user
+        */
+        [Test]
+        public void GetGSDTest()
```

Comment:
It might be better to name your tests the way they are named on the server for consistency's sake.